src/main/java/fakebot/task/ToDos.java

```diff
@@ -0,0 +1,17 @@
+package fakebot.task;
+/**
+ * ToDos Task Type
+ */
+public class ToDos extends Task {
```

Comment:
Perhaps you could have named the class TodoTask or just Todo instead of ToDos?

Reply:
will rename. thank you.